pkg/jsonmessage/jsonmessage.go

```diff
@@ -99,6 +99,7 @@ type JSONMessage struct {
 	ID              string        `json:"id,omitempty"`
 	From            string        `json:"from,omitempty"`
 	Time            int64         `json:"time,omitempty"`
+	TimeNano        int64         `json:"timeNano,omitempty"`
```

Comment:
Nah. It's straight forward and logical. Trying to overload the existing
field would be a mess.
On Sep 15, 2015 5:43 PM, "Alexander Morozov" <notifications@github.com>
wrote:

> In pkg/jsonmessage/jsonmessage.go
> <https://github.com/docker/docker/pull/16317#discussion_r39569479>:
>
> > @@ -99,6 +99,7 @@ type JSONMessage struct {
> >  	ID              string        `json:"id,omitempty"`
> >  	From            string        `json:"from,omitempty"`
> >  	Time            int64         `json:"time,omitempty"`
> > +	TimeNano        int64         `json:"timeNano,omitempty"`
>
> New field, looks sorta weird.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/16317/files#r39569479>.
>